### PR TITLE
Helm charts for Context App

### PR DESCRIPTION
**Pull request by olivakar**
There are some output files added to show the various combinations

### CODE REVIEW

File: samples/context-app-dotnet/http_NO_tls_BUT_sat.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    matchLabels:
+      app: context-app-deployment
+  template:
+    metadata:
+      labels:
+        app: context-app-deployment
+    spec:
+      containers:
+      - name: context-app-deployment
+        image: "dockeroliva/context-app-for-dss:x4"
```
Comment (timtay-microsoft) [inline samples/context-app-dotnet/http_NO_tls_BUT_sat.yaml:49]: We should probably rename this image to not use your name. Something like "iotoperations/context-app-for-dss:x4"?

Comment (timtay-microsoft) [reply samples/context-app-dotnet/http_NO_tls_BUT_sat.yaml:49]: Also, the x4 tag seems a bit odd. Maybe use "latest" as the tag?

"iotoperations/context-app-for-dss:latest"

File: samples/context-app-dotnet/ReadMe.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ Please refer to the official [MQTT broker documentation](https://learn.microsoft
 | `CLIENT_KEY_PASSWORD`      | The password for the client private key file.                      | string    | None                  | Yes (if TLS and auth are used)                        | x509-secret        |
```
Comment (timtay-microsoft) [inline samples/context-app-dotnet/ReadMe.md:192]: Any updates on this file to show how to package up the helm chart? I get that we don't expect to actually ship this package, but I gather we want to use this demo to show how a user would do something like that.

File: samples/context-app-dotnet/http_NO_tls_BUT_sat.yaml (hunk 1 above)
Comment (olivakar) [reply samples/context-app-dotnet/http_NO_tls_BUT_sat.yaml:49]: These files starting with http and sql are examples and only included in the PR to show what are the variations of output depending in the `values.yaml`.  These are not to be used directly. I actually included in my PR to show how they may look like. Probably should delete.

**Review by timtay-microsoft — APPROVED** [review]

**Pull request merged**